docs/codeql/writing-codeql-queries/creating-path-queries.rst

```diff
@@ -18,7 +18,7 @@ This topic provides information on how to structure a path query file so you can
 
     Note
 
-    The alerts generated by path queries are displayed by default in `LGTM <https://lgtm.com>`__ and included in the results generated using the :ref:`CodeQL CLI <codeql-cli>`. You can also view the path explanations generated by your path query `directly in LGTM <https://lgtm.com/help/lgtm/exploring-data-flow-paths>`__ or in the CodeQL :ref:`extension for VS Code <codeql-for-visual-studio-code>`.
+    The alerts generated by path queries are included in the results generated using the :ref:`CodeQL CLI <codeql-cli>`. You can also view the path explanations generated by your path query in the :ref:`CodeQL extension for VS Code <codeql-for-visual-studio-code>`.
```

Comment:
Should we mention code scanning here too?

Sadly it looks as if this is the closest thing we currently have to a good link for this information: [About code scanning alerts - GitHub Docs](https://docs.github.com/en/code-security/code-scanning/automatically-scanning-your-code-for-vulnerabilities-and-errors/about-code-scanning-alerts#about-alert-details).

Reply:
Good idea. I think that link is fine for now.